x-pack/plugins/apm/server/lib/helpers/setup_request.ts

```diff
@@ -108,12 +100,12 @@ export async function setupRequest<TParams extends SetupRequestParams>(
           )
         : undefined,
     config,
+    uiFiltersES: decodeUiFilters(query.uiFilters),
```

Comment:
Seems like it was never actually optional! Always defaulted to `[]`

Reply:
it was optionally spread into the core setup object, but yes `decodeUiFilters` was always invoked regardless.